feature-libs/product-configurator/rulebased/components/attribute/types/base/configurator-attribute-multi-selection-base.component.ts

```diff
@@ -99,4 +100,40 @@ export abstract class ConfiguratorAttributeMultiSelectionBaseComponent extends C
 
     this.selectionChange.emit(event);
   }
+
+  /**
+   * Extract corresponding price formula parameters
+   *
+   * @return {ConfiguratorPriceComponentOptions} - New price formula
+   */
+  extractPriceFormulaParameters(): ConfiguratorPriceComponentOptions {
+    return {
+      quantity: 0,
+      price: {
+        value: 0,
+        currencyIso: '',
+      },
+      priceTotal: this.attribute?.attributePriceTotal,
```

Comment:
```suggestion
      priceTotal: this.attribute.attributePriceTotal,
```
Attribute is mandatory

Reply:
`attribute` is mandatory but `attributePriceTotal` is not mandatory
the question mark is for `attributePriceTotal`

Reply:
My point is:` this.attribute.attributePriceTotal `is fine. No need to add '?', as this refers to this.attribute.
I know that attributePriceTotal is optional.

Reply:
Done!